### PR TITLE
API provides functionality for user to see their current order

### PR DESCRIPTION
**Pull request by DavDan1**
PT story: https://www.pivotaltracker.com/story/show/177903774

### User story
```
As a restaurant API
In order to give the user a way to see their current order
I would like to serve the client with the current pending order for that user
```
### Suggested changes
- Adds functionality for user to see the current order

### CODE REVIEW

File: config/routes.rb [hunk 1]
```diff
@@ -2,6 +2,6 @@
   mount_devise_token_auth_for 'User', at: 'api/auth'
   namespace :api do
     resources :menu_items, only: [:index]
-    resources :orders, only: %i[create update]
+    resources :orders, only: %i[create update show]
```
Comment (1CIM) [inline config/routes.rb:5]: can you explain what the %i dose?

Comment (DavDan1) [reply config/routes.rb:5]: it removes the need of commas inside of array.

File: spec/models/order_item_spec.rb [hunk 2]
```diff
@@ -1,5 +1,3 @@
-require 'rails_helper'
```
Comment (christerforsgren91) [inline spec/models/order_item_spec.rb:1]: Why did you remove this?

File: spec/requests/api/api_provides_user_to_create_an_order_spec.rb [hunk 3]
```diff
@@ -27,4 +27,22 @@
...
+        params: { 
+          menu_item_id: menu_item.id
+        },
+        headers: {}
+    end
+    it 'is expected to return 401 status' do
+      expect(response).to have_http_status 401
+    end
+    it 'is expected to show a message' do
+      expect(response_json['errors'].first).to eq "You need to sign in or sign up before continuing."
```
Comment (1CIM) [inline spec/requests/api/api_provides_user_to_create_an_order_spec.rb]: can't you use single quotes here? so it's all the same

**Review by christerforsgren91 — COMMENTED** [review]
Looks good boys! Keep it up

File: spec/requests/api/api_provides_user_to_create_an_order_spec.rb (hunk 3 above)
Comment (DavDan1) [reply spec/requests/api/api_provides_user_to_create_an_order_spec.rb]: Yes, thank you , I will resolve this issue right away!

**Review by 1CIM — APPROVED** [review]

File: spec/models/order_item_spec.rb (hunk 2 above)
Comment (lukeperrin10) [reply spec/models/order_item_spec.rb:1]: When a model is created using the rails generator, this is automatically added to the models in the spec file. This is not needed in the file as this is called on globally.

Comment (christerforsgren91) [reply spec/models/order_item_spec.rb:1]: okey, thanks!
